test/functional/test_framework/test_framework.py

```diff
@@ -203,67 +197,60 @@ def main(self):
 
     # Public helper methods. These can be accessed by the subclass test scripts.
 
-    def start_node(self, i, dirname, extra_args=None, rpchost=None, timewait=None, binary=None, stderr=None):
+    def start_node(self, i, dirname, extra_args=[], rpchost=None, timewait=None, binary=None, stderr=None):
```

Comment:
Nice. Thanks @kallewoof , I wasn't aware of that, but it makes perfect sense.

Changed the default argument to `None` as suggested.